packages/devtools_app_shared/lib/src/service/service_manager.dart

```diff
@@ -269,6 +272,20 @@ class ServiceManager<T extends VmService> {
     _connectedState.value = connectionState;
   }
 
+  Future<void> _configureIsolateSettings() async {
+    if (service != null) {
+      try {
+        final vmService = service!;
+        await vmService.setFlag('pause_isolates_on_start', 'true');
+        await vmService.requirePermissionToResume(
+          onPauseStart: true,
```

Comment:
@bkonyi do we need `onPauseReload` here as well? What is the difference between them? (Is `onPauseStart` for hot-restarts and `onPauseReload` for hot-reloads?)

Reply:
Ok sounds good! This is just for hot-restart, not hot-reload.